site/en/docs/devtools/console/reference/index.md

```diff
@@ -84,13 +72,11 @@ The links below explain each setting:
 ### Open the Console Sidebar {: #sidebar }
 
```

Comment:
Add explanation here on how is it different from the filter dropdown and its use case. 

Filter dropdown: If you want to filter changes more permanently across pages.
Sidebar: If you want to filter just for a short while. E.g. You are debugging a specific issue on a page. Temporarily filter with side bar until you resolve the issue.